mDeepFRI/cli.py

```diff
@@ -0,0 +1,241 @@
+import logging
+from pathlib import Path
+
+import click
+
+from mDeepFRI import __version__
+from mDeepFRI.database import build_database
+from mDeepFRI.pipeline import metagenomic_deepfri
+from mDeepFRI.utils.utils import download_model_weights
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+@click.group()
+@click.option("--debug/--no-debug", default=False)
+@click.version_option(version=__version__)
+@click.pass_context
+def main(ctx, debug):
```

Comment:
I'm more used to main function definition to be placed at the bottom of the file